eng/pipelines/eng-common-sync.yml

```diff
@@ -92,6 +92,7 @@ stages:
           environment: githubmerges
 
           pool:
+            name: azsdk-pool-mms-win-2022-general
```

Comment:
Can we try switching to ubuntu here?

Reply:
Yes it theory but we would need to test. We did move the first job to ubuntu.